dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hdfs/ScanContext.java

```diff
@@ -45,10 +55,15 @@ private enum CsvHeader {
     Owner,
     Group,
     Permission,
+    ModificationTime,
+    NumberOfFilesAndSubdirs,
+    NumberOfSubdirs,
+    StoragePolicy,
   }
 
   ScanContext(FileSystem fs, Writer outputSink) throws IOException {
     this.fs = fs;
+    this.dfsClient = ((DistributedFileSystem) fs).getClient();
```

Comment:
Consider checking this before creating a `ScanContext` in `HdfsPermissionExtractionTask`

Reply:
Done.